http/legacy/write_handler_test.go

```diff
@@ -94,7 +93,7 @@ func TestWriteHandler_BucketAndMappingExists(t *testing.T) {
 	handler := NewWriterHandler(&PointsWriterBackend{
 		HTTPErrorHandler:   DefaultErrorHandler,
 		Logger:             zaptest.NewLogger(t),
-		BucketService:      authorizer.NewBucketService(bucketService),
+		BucketService:      bucketService,
```

Comment:
I believe this accurately reflects the setup constructed by the `influxd` launcher. It'd be nice to have one setup function that both the tests and main code could call, to ensure they stay in sync.